runtime/themes/everforest_dark.toml

```diff
@@ -100,6 +100,11 @@
 "ui.virtual.indent-guide" = { fg = "bg4" }
 "ui.virtual.inlay-hint" = { fg = "grey0" }
 "ui.virtual.wrap" = { fg = "grey0" }
+"ui.virtual.ruler" = { bg = "bg3" }
```

Comment:
this theme key has been added on master too so you need to remove it here or the theme breaks

Reply:
Oh how did that crawl in haha, thanks!